client/state/lib/middleware.js

```diff
@@ -91,6 +95,22 @@ const updateSelectedSiteForDesktop = ( dispatch, action, getState ) => {
 	desktop.setSelectedSite( selectedSite );
 };
 
+/*
+ * Here be dragons.
+ */
```

Comment:
But where is the dragon emoji? 🐉 

Maybe add some more background context in the comment